extensions/cache/src/main/java/org/radargun/stages/cache/background/BackgroundStressorLogic.java

```diff
@@ -19,7 +19,7 @@
  * @author Radim Vansa &lt;rvansa@redhat.com&gt;
  * @author Michal Linhard &lt;mlinhard@redhat.com&gt;
  */
-class LegacyLogic extends AbstractLogic {
+class BackgroundStressorLogic extends AbstractLogic {
```

Comment:
OK. But I don't see a reason why anything should have "Legacy" in its name. So let's take is as a step to better readability of this class.